src/main/java/org/sagebionetworks/bridge/spring/controllers/FPHSController.java

```diff
@@ -56,7 +56,7 @@ public StatusMessage registerExternalIdentifier() {
         UserSession session = getAuthenticatedSession();
         
         ExternalIdentifier externalId = parseJson(ExternalIdentifier.class);
-        fphsService.registerExternalIdentifier(session.getAppId(), session.getHealthCode(), externalId);
+        fphsService.registerExternalIdentifier(session.getAppId(), session.getId(), externalId);
```

Comment:
Check or revert - we're moving this over to ID, is that happening at every tier?

Reply:
reverted